tests/artist-create.test.js

```diff
@@ -0,0 +1,27 @@
+const { expect } = require('chai');
+ const request = require('supertest');
+ const db = require('../src/db/index');
+ const app = require('../src/app');
+
+ describe('create artist', () => {
+   describe('/artists', () => {
+     describe('POST', () => {
+       it('creates a new artist in the database', async () => {
```

Comment:
Could another test be added here to confirm what would happen when trying to add an artist that is already in the database?